utils/generate-commands-json.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+'''
```

Comment:
I usually use `"""`

Reply:
Is it better than ```'''```? Different? PEP8 concesus?

Reply:
NM, https://www.python.org/dev/peps/pep-0257/